res/database/Default/database_news.xml

```diff
@@ -2419,6 +2419,9 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 			</description>
 			<data genre="5" price="1.0" quality="49" />
 			<availability year_range_from="-1" year_range_to="-1" />
+			<effects>
+				<effect trigger="happen" type="triggernews" time="1,8,16" news="88e366d7-7bca-4598-bab4-e19eeeb966a8" />
```

Comment:
it exists, in the new `scr0llbaer-constr-news-2024-08.xml` file. Did you add the new files / refresh project?

as described somewhere in that file, curiously, along with some additional standalone news items there existed additional news items for some threads in a previous version of TVTower, but they fell by the wayside, perhaps because of some database migration, perhaps deliberately due to lack of quality (they may be not *great*, but not *that* bad either).

They're "new" items, hence for now in that extra file, but later to be moved here.

Reply:
OK, I will check again. I thought when I manually searched for the ID, only the trigger was found.